docker/hive2-kerberos/Dockerfile

```diff
@@ -81,9 +81,22 @@ RUN keytool -genkeypair \
     -dname "CN=presto-master, OU=, O=, L=, S=, C="
 RUN chmod 644 /etc/presto/conf/keystore.jks
 
-# CREATE DB USER ACCOUNT
-RUN /usr/sbin/kadmin.local -q "addprinc -randkey db-user/localhost@KEBOOLA.COM" && \
-    /usr/sbin/kadmin.local -q "xst -norandkey -k /root/kerberos-keytabs/db-user.keytab db-user/localhost"
+# CREATE SSL FOR HIVE
+RUN keytool -genkey \
```

Comment:
Pre Kerberos instanciu som zapol SSL, kedze tak to ma CSAS + pridal som podporu do PHP.